sync/src/tasks/tests.rs

```diff
@@ -105,6 +105,9 @@ pub async fn test_sync_invalid_target() -> Result<()> {
     Ok(())
 }
 
+#[ignore = "This test is for the scenario that a block failed to connect to the main will be stored in the \
+failure storage and the sync will return failure instantly the next time the block shows up again, \
+which is no longer suitable for the dag"]
```

Comment:
Change the networkid to network with u64::max dag fork height, such as Test. This test can be fixed.

Reply:
It cannot be fixed as you said. You try and will know.

Reply:
emm... It did failed. This a case for the single chain, and it should be fixed by this walkaround solution. Could some updates break the procedures? But anyway, it's your call.